src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs

```diff
@@ -395,25 +395,26 @@ private ISet<DocumentId> GetProjectOpenDocuments_NoLock(ProjectId project)
             return openDocs;
         }
 
-        protected internal void OnDocumentOpened(
+        internal Task OnDocumentOpenedAsync(
             DocumentId documentId, SourceTextContainer textContainer,
```

Comment:
Yes.  This is to revert the issue *temporarily* to unblock Jenkins.  We can then restore the async behavior next week once we can get it working properly.

I don't want to have people affected by this issue for the next 5 days over thanksgiving.